test/Libraries/RevitIntegrationTests/SelectionTests.cs

```diff
@@ -612,6 +613,115 @@ public void SelectElementsByCategoryInView()
+            var outputDictionary = GetPreviewValue(selectNode.GUID.ToString());
+            Assert.NotNull(outputDictionary);
+
+            var placedRooms = GetPreviewCollection("e512c62255eb4a349ce7126af54ef03d");
+            var unplacedRooms = GetPreviewCollection("ae185814931b4acfbe92d38a9abfc79b");
+            var notEnclosedRooms = GetPreviewCollection("b3959f94df634a3787ae421dc8e5f409");
+            var redundantRooms = GetPreviewCollection("bc195cf9592b4e7bb8574414ae149f1b");
+
+            Assert.NotNull(placedRooms);
+            Assert.NotNull(unplacedRooms);
+            Assert.NotNull(notEnclosedRooms);
+            Assert.NotNull(redundantRooms);
+
+            AssertElementCategoryOnCollection(placedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(unplacedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(notEnclosedRooms, expectedElementsCategoryString);
+            AssertElementCategoryOnCollection(redundantRooms, expectedElementsCategoryString);
+
+        }
+
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void SelectModelElementById()
+        {
+            OpenAndAssertNoDummyNodes(Path.Combine(workingDirectory, @".\Selection\SelectModelElementById.dyn"));
+
+            RunCurrentModel();
+
+            var selectNode =
+                ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<RevitNodeModel>();
+            Assert.NotNull(selectNode);
+
+            var expectedId = 184176;
+            var element = GetPreviewValue(selectNode.GUID.ToString()) as Revit.Elements.Element;
+            Assert.NotNull(element);
+            Assert.AreEqual(expectedId, element.Id);
+        }
+
+
+        [Test]
+        [TestModel(@".\SampleModel.rvt")]
+        public void RoofTypesDropDown()
+        {
+            OpenAndAssertNoDummyNodes(Path.Combine(workingDirectory, @".\Selection\RoofTypesDropDown.dyn"));
+
+            RunCurrentModel();
+
+            var selectNode =
+                ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<RevitDropDownBase>();
+            Assert.NotNull(selectNode);
+
+            var element = GetPreviewValue(selectNode.GUID.ToString()) as Revit.Elements.RoofType;
+            Assert.NotNull(element);
+        }
+
+        [Test]
+        [TestModel(@".\SampleModel.rvt")]
+        public void SheetsDropDown()
+        {
+            OpenAndAssertNoDummyNodes(Path.Combine(workingDirectory, @".\Selection\SheetsDropDown.dyn"));
+
+            RunCurrentModel();
+
+            var selectNode =
+                ViewModel.Model.CurrentWorkspace.FirstNodeFromWorkspace<RevitDropDownBase>();
+            Assert.NotNull(selectNode);
+
+            var element = GetPreviewValue(selectNode.GUID.ToString()) as Revit.Elements.Views.Sheet;
+            Assert.NotNull(element);
+        }
+
+        [Test]
+        [TestModel(@".\SampleModel.rvt")]
+        public void ViewFamilyTypesDropDown()
```

Comment:
Hi @SHKnudsen , for your this test, I can't find any new nodes called "ViewFamilyTypes" in this PR. Did you miss this node?

Reply:
looks like i accidentally removed it when merging in master. Should be resolved now.